pelicanconf.py

```diff
@@ -21,7 +21,7 @@
 AUTHOR_FEED_RSS = None
 
 ARTICLE_PATHS = ["blog"]
-THEME = "themes/inpycon2021"
+THEME = "themes/inpycon2023"
 
 # Blogroll
 LINKS = (("InPyCon", "https://in.pycon.org/"), ("PSSI", "http://pssi.org.in/"))
```

Comment:
I'm not sure where these links are used, but if PSSI is not active anymore, we should remove this.

Reply:
I am not sure, so I left as it is. Let me add a note and we'll remove it when we update the final theme for Blog 2023.